include/odp/api/spec/ipsec.h

```diff
@@ -259,6 +259,10 @@ typedef struct odp_ipsec_capability_t {
 	 */
 	odp_support_t retain_header;
 
+	/** Implementation will automatically truncate TFC padding in received
+	 *  packets */
+	odp_bool tfc_padding_truncate;
```

Comment:
Why not `odp_support_t` same as other capabilities here?

Reply:
Because it's not a declaration of supporting a feature but rather a statement how implementation handles this case. It is not possible to switch it. So I preferred to have odp_bool_t here.

Reply:
Some implementations may be able to configure this behavior in `odp_ipsec_config()`, in which case `odp_support_t` would be useful. We can make that change later, but it just looks more consistent here. Not a big deal, however.